source/daplink/cmsis-dap/DAP.h

```diff
@@ -101,6 +103,104 @@
+#define ID_DAP_Vendor64                 0xC0U
+#define ID_DAP_Vendor65                 0xC1U
+#define ID_DAP_Vendor66                 0xC2U
+#define ID_DAP_Vendor67                 0xC3U
+#define ID_DAP_Vendor68                 0xC4U
+#define ID_DAP_Vendor69                 0xC5U
+#define ID_DAP_Vendor70                 0xC6U
+#define ID_DAP_Vendor71                 0xC7U
+#define ID_DAP_Vendor72                 0xC8U
+#define ID_DAP_Vendor73                 0xC9U
+#define ID_DAP_Vendor74                 0xCAU
+#define ID_DAP_Vendor75                 0xCBU
+#define ID_DAP_Vendor76                 0xCCU
+#define ID_DAP_Vendor77                 0xCDU
+#define ID_DAP_Vendor78                 0xCEU
+#define ID_DAP_Vendor79                 0xCFU
+#define ID_DAP_Vendor80                 0xD0U
+#define ID_DAP_Vendor81                 0xD1U
+#define ID_DAP_Vendor82                 0xD2U
+#define ID_DAP_Vendor83                 0xD3U
+#define ID_DAP_Vendor84                 0xD4U
+#define ID_DAP_Vendor85                 0xD5U
+#define ID_DAP_Vendor86                 0xD6U
+#define ID_DAP_Vendor87                 0xD7U
+#define ID_DAP_Vendor88                 0xD8U
+#define ID_DAP_Vendor89                 0xD9U
+#define ID_DAP_Vendor90                 0xDAU
+#define ID_DAP_Vendor91                 0xDBU
+#define ID_DAP_Vendor92                 0xDCU
+#define ID_DAP_Vendor93                 0xDDU
+#define ID_DAP_Vendor94                 0xDEU
+#define ID_DAP_Vendor95                 0xDFU
+#define ID_DAP_Vendor96                 0xE0U
+#define ID_DAP_Vendor97                 0xE1U
+#define ID_DAP_Vendor98                 0xE2U
+#define ID_DAP_Vendor99                 0xE3U
+#define ID_DAP_Vendor100                0xE4U
+#define ID_DAP_Vendor101                0xE5U
+#define ID_DAP_Vendor102                0xE6U
+#define ID_DAP_Vendor103                0xE7U
+#define ID_DAP_Vendor104                0xE8U
+#define ID_DAP_Vendor105                0xE9U
+#define ID_DAP_Vendor106                0xEAU
+#define ID_DAP_Vendor107                0xEBU
+#define ID_DAP_Vendor108                0xECU
+#define ID_DAP_Vendor109                0xEDU
+#define ID_DAP_Vendor110                0xEEU
+#define ID_DAP_Vendor111                0xEFU
+#define ID_DAP_Vendor112                0xF0U
+#define ID_DAP_Vendor113                0xF1U
+#define ID_DAP_Vendor114                0xF2U
+#define ID_DAP_Vendor115                0xF3U
+#define ID_DAP_Vendor116                0xF4U
+#define ID_DAP_Vendor117                0xF5U
+#define ID_DAP_Vendor118                0xF6U
+#define ID_DAP_Vendor119                0xF7U
+#define ID_DAP_Vendor120                0xF8U
+#define ID_DAP_Vendor121                0xF9U
+#define ID_DAP_Vendor122                0xFAU
+#define ID_DAP_Vendor123                0xFBU
+#define ID_DAP_Vendor124                0xFCU
+#define ID_DAP_Vendor125                0xFDU
+#define ID_DAP_Vendor126                0xFEU
```

Comment:
I'm not sure we need to predefine all these command IDs. It might be better to define only `ID_DAP_VendorExFirst` and `ID_DAP_VendorExLast` so the range is documented in the code. (Note the "Ex" in the name to match `DAP_ProcessVendorCommandEx()`.) Then any vendor commands should have their own definitions in a HIC-specific header file, something like `#define ID_DAP_CypressFoo (ID_DAP_VendorExFirst + 1)

Reply:
I've made changes according to your comments.